cactusbot/packets/message.py

```diff
@@ -313,7 +313,7 @@ def split(self, seperator=' ', maximum=None):
 
         Parameters
         ----------
-        seperator : :obj:`str`, default `' '`
+        separator : :obj:`str`, default `' '`
```

Comment:
Ooh, good catch! That always gets me. :stuck_out_tongue_winking_eye: 

Can you change the actual variable name too, though? (In the method declaration above, and in the code below.) That way, the docs are actually accurate.